stdlib/3.4/asyncio/queues.pyi

```diff
@@ -32,15 +33,19 @@ class Queue(Generic[T]):
     @coroutine
     def get(self) -> Future[T]: ...
     def get_nowait(self) -> T: ...
+    if sys.version_info >= (3, 4, 4):
```

Comment:
This version check does not actually work with mypy, since mypy only lets you supply the first two digits of the version (i.e. `(3, 4)`). We have to make up our mind -- do we consider this valid for any version >= 3.4 (thus lying about 3.4.0--3.4.3) or for any version >= 3.5 (thus lying about 3.4.4)?